docs/nemoretriever-ocr.md

```
@@ -45,16 +45,15 @@ For more information about NeMo Retriever OCR, refer to [NeMo Retriever OCR v1 C
-3. **Stop Paddle OCR deployment if already running**:
-   ```bash
-   USERID=$(id -u) docker compose -f deploy/compose/nims.yaml down paddle
-   ```
-
-4. **Deploy NeMo Retriever OCR Service**:
+3. **Deploy NeMo Retriever OCR Service** (now default):
    ```bash
    USERID=$(id -u) docker compose -f deploy/compose/nims.yaml --profile nemoretriever-ocr up -d
    ```
 
+   :::{note}
```

markdownlint-cli2 (0.18.1)

Unordered list indentation
Expected: 2; Actual: 4

(MD007, ul-indent)